app/main.py

```diff
@@ -1,8 +1,25 @@
 class Person:
-    # write your code here
-    pass
+    people: dict = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    person_list = [Person(human["name"], human["age"]) for human in people]
+    dict_for_merge = {el.name: el for el in person_list}
+    Person.people |= dict_for_merge
```

Comment:
The use of the `|=` operator to update the `Person.people` dictionary is correct if you are using Python 3.9 or later. Ensure that your environment supports this syntax. If not, consider using the `update()` method instead.